django_project/minisass_authentication/serializers.py

```diff
@@ -138,6 +138,12 @@ def validate_password(self, value):
         if all(remaining_requirements.values()):
             missing_criteria = ', '.join([key for key, val in remaining_requirements.items() if val])
             raise serializers.ValidationError(f'Missing password criteria: {missing_criteria}')
+
+        is_password_used = PasswordHistory.is_password_used(
+            self.context['user'], value
+        )
+        if is_password_used:
+            raise serializers.ValidationError('Password has been used')
```

Comment:
This password has been used before. Please choose a new and unique password.

Oops! The password you entered has been used before. Kindly select a different password for better security.

would these be more informative ?

Reply:
I'll update to this

> This password has been used before. Please choose a new and unique password.

Reply:
Updated in: https://github.com/kartoza/miniSASS/pull/570